react/features/base/config/functions.web.ts

```diff
@@ -60,3 +60,44 @@ export function areAudioLevelsEnabled(state: IReduxState): boolean {
     // Default to false for React Native as audio levels are of no interest to the mobile app.
     return navigator.product !== 'ReactNative' && !state['features/base/config'].disableAudioLevels;
 }
+
+/**
+ * Sets the defaults for deeplinking.
+ *
+ * @param {IDeeplinkingConfig} deeplinking - The deeplinking config.
+ * @returns {void}
+ */
+export function _setDeeplinkingDefaults(deeplinking: IDeeplinkingConfig) {
+    const { desktop, android, ios } = deeplinking;
+
+    if (desktop) {
```

Comment:
Shouldn't we add the section if missing?

Reply:
yeah realised that afterwards and did a commit with this.